Keras_Tensorflow/04_DeployOnAKS.ipynb

```diff
@@ -73,33 +72,74 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mNote, we have launched a browser for you to login. For old experience with device code, use \"az login --use-device-code\"\u001b[0m\n",
+      "\n",
+      "(firefox:82222): LIBDBUSMENU-GLIB-WARNING **: Unable to get session bus: Error spawning command line 'dbus-launch --autolaunch=e05d81ec9eea4cb7abfd3fd9ec6f3655 --binary-syntax --close-stderr': Child process exited with code 1\n",
+      "\u001b[33mYou have logged in. Now let us find all subscriptions you have access to...\u001b[0m\n",
```

Comment:
@marabout2015 We need to clean the output here. You can just comment the code and run the cell and then uncomment.